### PR TITLE
Exams by Viktoriia Kapys, IM-22

### PR DESCRIPTION
**Pull request by viktoriiakapys**
I changed files "day.js" and "between.js".

### CODE REVIEW

**Comment by kushnirko** [comment]
### Good job! A great idea was to split condition using `slice()` (file **day.js**). But to make code in file **between.js** easier to understand I'd rename variable` lastString` to `substring`.

**Review by kushnirko — APPROVED** [review]
LGTM!

**Review by yatsenkoM — APPROVED** [review]
Good job with the code! I like the changes you made, the code really looks better

File: Tasks/day.js [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
-_parse_day_ = (s) => {
-  let i;
-  for (i = 0; i < D.length; i++) {
-    if (s.startsWith(D[i].toLowerCase())) {
-      return i + 1;
+const parseDay = (currentDay) => {
+  for (const sameDay of days) {
+    const index = days.indexOf(sameDay);
+    const smallLetters = sameDay.toLowerCase();
+    const cutWord = currentDay.slice(0, 3);
```
Comment (tshemsedinov) [inline Tasks/day.js:18]: Hardcoded magic number

File: Tasks/between.js [hunk 2]
```diff
@@ -1,21 +1,13 @@
...
-      i = str.indexOf(s);
-      if (i === -1) {
-        return '';
-      } else {
-        str = str.substring(0, i);
-      }
-    }
-  }
-  return str;
+const getValueBetween = (string, firstSymbol, secondSymbol) => {
```
Comment (tshemsedinov) [inline Tasks/between.js:4]: Not a simbol but substring

**Pull request closed without merging**